merqube_client_lib/api_client/base.py

```diff
@@ -108,8 +108,7 @@ def get_index_defs(
         """
 
         endpoint = "/index"
-        if include_nonprod:
-            endpoint += "?type=all"
+        endpoint += "?stage=all" if include_nonprod else "?stage=prod"
```

Comment:
Don't we not need `?stage=all` if `include_nonprod`?

Might be cleaner to do:
`if include_nonprod`
` endpoint += "?stage=prod`

Reply:
yes, could remove that (harmless but will remove)